automation-tools/rpkg.md

```diff
@@ -0,0 +1,52 @@
+# RPKG
```

Comment:
```suggestion
# RPKG-UTIL
```
rpkg is a different project :D why not make things more complicated, right?

Reply:
I've added both..;-)